projects/driver_controls/src/pedal_fsm.c

```diff
@@ -0,0 +1,81 @@
+#include "pedal_fsm.h"
+#include "input_event.h"
+#include "event_arbiter.h"
+#include "log.h"
+
+// Pedal FSM state definitions
+
+FSM_DECLARE_STATE(state_brake);
+FSM_DECLARE_STATE(state_coast);
+FSM_DECLARE_STATE(state_driving);
+FSM_DECLARE_STATE(state_cruise_control);
+
+// Pedal FSM transition table definitions
+
+FSM_STATE_TRANSITION(state_brake) {
+  FSM_ADD_TRANSITION(INPUT_EVENT_PEDAL_COAST, state_coast);
```

Comment:
Don't you need to add transitions back to itself to handle the repeated events for analog states?